app/test/integration_test.go

```diff
@@ -249,7 +249,7 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 			"medium random with timeout height",
 			mustNewBlob(ns1, tmrand.Bytes(100000), appconsts.ShareVersionZero),
 			[]user.TxOption{
-				user.SetTimeoutHeight(1000),
+				user.SetTimeoutHeight(10000),
```

Comment:
The configuration is fixed in this PR as to actually use a default of 1 ms for a timeout commit. We have to either increase this since blocks are produced super fast, or add in a delay to slow block production. 

I'll defer to reviewers if they have an opinion. Adding a timeout commit also seems like a great option, since we have no real benefit of producing over 1000 blocks in a single test lol